sql/src/test/java/io/crate/execution/dml/upsert/SourceFromCellsTest.java

```diff
@@ -63,6 +67,7 @@ public void setUpExecutor() throws Exception {
         e = SQLExecutor.builder(clusterService)
             .addTable("create table t1 (x int, y int, z as x + y)")
             .addTable("create table t2 (obj object as (a int, c as obj['a'] + 3), b as obj['a'] + 1)")
+            .addPartitionedTable("create table t3 (p int not null) partitioned by (p)")
```

Comment:
does it make sense to add another case with a composite partition by clause where one field can be null and the other not null? O perhaps an integration test makes more sense for this case (if it does not exist already)?

Reply:
Not sure what that scenario would cover in addition.